jboss-eap-clustered-rhel7/README.md

```diff
@@ -106,9 +104,9 @@ Build your environment with EAP 7.2 on top of RHEL 8.0 on Azure in a few simple
     
    - **SSH Key Data** - Generate an SSH key using Puttygen and provide the data here.
    
-   - **VM Size** - Choose the size of the VM from the options in dropdown
+   - **VM Size** - Choose the appropriate size of the VM from the dropdown options. 
 
-   - Leave the rest of the Parameter Value as it is and proceed.
+   - Leave the rest of the Parameter Values (artifacts and Location) as is and proceed to purchase. 
```

Comment:
Modified this line as follows. Please suggest

Leave the rest of the parameter values (artifacts and Location) as is and accept the terms and conditions before clicking on Purchase